src/pages/Main/Main.js

```diff
@@ -1,8 +1,87 @@
-import React from 'react';
+import React, { useState, useRef, useEffect } from 'react';
+import { MAIN_CATEGORY } from './mainCategoryData';
 import './Main.scss';
 
 const Main = () => {
-  return <div>Main</div>;
+  const sectionRef = useRef(null);
+  const [next, setNext] = useState(0);
+  const carouselButtonCount = 3;
+
+  const nextButton = e => {
+    if (next >= carouselButtonCount) {
+      setNext(3);
+    } else {
+      setNext(next + 1);
+    }
+  };
+  const prevButton = e => {
+    if (next > carouselButtonCount) {
+      setNext(3);
+    } else {
+      setNext(next - 1);
+    }
+    if (next < 1) {
+      setNext(0);
+    }
+  };
```

Comment:
```suggestion
  const handleSlideBtn = value => {
    if (next + value === carouselButtonCount + 1 || next + value === -1) return;
    setNext(prev => prev + value);
  };
```

이 두가지 함수는 하나의 함수로 합칠 수 있고, 사용하는 곳에서 담아 주는 인자 값에 따라서
다른 동작이 구현되도록 할 수 있을 것 같네요!

Reply:
수정사항 리뷰 감사합니다!